frontend/src/components/auth/Signup/index.tsx

```diff
@@ -51,6 +64,17 @@ const Signup = () => {
           navigation={navigation}
         />
       );
+    case "terms conditions":
```

Comment:
not sure if this is the best way to handle "skipping" the terms and conditions page for donors... any suggestions/feedback is appreciated!

Reply:
see comment below